### PR TITLE
Applied tag color in session list

### LINKED ISSUE #541: Apply tag color for category
## Kind (Required)

- Other

 ## Overview (Required)

- 
image

no | category | background | textcolor
-- | -- | -- | --
1 | UI・デザイン (UI and Design) | #E5E8EA | #353435
2 | アプリアーキテクチャ (Designing App Architecture) | #E5E8EA | #353435
3 | Androidプラットフォーム (Android Platforms) | #E5E8EA | #353435
4 | 保守・運用・テスト (Maintenance Operations and Testing) | #E5E8EA | #353435
5 | 開発体制 (Development processes) | #E5E8EA | #353435
6 | Android FrameworkとJetpack(Support Library) (Android Framework and Jetpack (a.k.a. Support Library)) | #E5E8EA | #353435
7 | 開発ツール (Productivity and Tools) | #E5E8EA | #353435
8 | クロスプラットフォーム (Cross-platform Development) | #E5E8EA | #353435
9 | xR, | #E5E8EA | #353435
10 | セキュリティ (Security), | #E5E8EA | #353435
11 | ハードウェア (Hardware) | #E5E8EA | #353435
12 | その他 (Other) | #E5E8EA | #353435
  |   |   |  
A | 日本語 | #FDD7D4 | #F63529
B | 英語 | #E8E6F3 | #190089
  |   |   |  
  | 初心者歓迎 | #C1EBDD | #00BE80
  | ルーム | #D8D7DC | #353435

 ## Links	

-

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #541 

## Overview (Required)
- Applied tags color in session list.
Change `language` of `SpeechSession` to use `Lang`.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
🆒

File: feature/session/src/main/res/layout/item_session.xml [hunk 1]
```diff
@@ -125,46 +127,46 @@
...
                 android:layout_height="wrap_content"
                 android:clickable="false"
                 android:ellipsize="end"
                 android:enabled="false"
-                android:text="@string/session_simultaneous_interpretation"
+                android:text="@string/session_for_beginners"
                 android:textAppearance="@style/TextAppearance.App.Caption"
-                android:textColor="@color/white"
-                app:chipBackgroundColor="#7982e1"
+                android:textColor="#00BE80"
```
Comment (takahirom) [inline feature/session/src/main/res/layout/item_session.xml]: Please create `feature/session/src/main/res/values/colors.xml` And Can you declare it in `feature/session/src/main/res/values/colors.xml` ?
like `session_english_talk_text` 🙏

File: data/repository-impl/src/main/java/io/github/droidkaigi/confsched2019/data/repository/mapper/SessionMappers.kt [hunk 2]
```diff
@@ -4,12 +4,7 @@ import com.soywiz.klock.DateTime
 import com.soywiz.klock.hours
 import io.github.droidkaigi.confsched2019.data.db.entity.SessionWithSpeakers
 import io.github.droidkaigi.confsched2019.data.db.entity.SpeakerEntity
-import io.github.droidkaigi.confsched2019.model.Category
-import io.github.droidkaigi.confsched2019.model.LocaledString
-import io.github.droidkaigi.confsched2019.model.Room
-import io.github.droidkaigi.confsched2019.model.Session
-import io.github.droidkaigi.confsched2019.model.SessionType
-import io.github.droidkaigi.confsched2019.model.Speaker
+import io.github.droidkaigi.confsched2019.model.*
```
[the remark this answers is not in the document]
Comment (NUmeroAndDev) [reply data/repository-impl/src/main/java/io/github/droidkaigi/confsched2019/data/repository/mapper/SessionMappers.kt]: I fixed it 🙇

**Comment by NUmeroAndDev** [comment]
@takahirom 
>Please create feature/session/src/main/res/values/colors.xml And Can you declare it in feature/session/src/main/res/values/colors.xml ?
like session_english_talk_text 🙏

I declared it to colors.xml.
Please check again 🙏

**Pull request merged**